docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -11,46 +11,64 @@ other resources such as Kubernetes clusters and applications coming in the futur
 
 To upgrade Teleport Connect to a newer version, drag the new version to the Applications folder.
+</TabItem>
+<TabItem label="Linux">
+Download the DEB (Debian-based distros) or RPM (RHEL-based distros) package and install it using
+your package manager. Repeat the process for in-place upgrades.
+
+You can also download the project as a `tar.gz` file to extract and run it in place.
+</TabItem>
+<TabItem label="Windows">
+Download and run the installer `.exe` file. It will install and open Teleport Connect without
+further user input.
+
+Repeat the process with newer versions to upgrade.
+</TabItem>
+<TabItem label="From Source">
+</TabItem>
+</Tabs>
 
 ## User interface
 
 ![user interface of Teleport Connect](../../img/use-teleport/connect-ui-overview.png)
 
 The top bar of Teleport Connect consists of:
 
-- The **profile selector** (the top right), which allows you to switch between profiles on different Teleport
-  clusters as well as log in or out of the clusters.
+- The **profile selector** (the top right), which allows you to switch between profiles on different
+  Teleport clusters as well as log in or out of the clusters.
 - The **connection list** (the top left) showing recent connections, allowing you to seamlessly switch
   between them.
 - The **command bar** (in the middle), which launches any command you input in a new terminal tab.
-- The **cluster selector** (to the left of the command bar), which shows up only if you have set up Trusted Clusters and there are leaf clusters
-  connected to the root cluster. It lets you browse leaf cluster resources. It also changes
-  which cluster the commands in the command bar are going to target.
+- The **cluster selector** (to the left of the command bar), which shows up only if you have set up
+  Trusted Clusters and there are leaf clusters connected to the root cluster. It lets you browse
+  leaf cluster resources. It also changes which cluster the commands in the command bar are going to target.
 
 ## Connecting to an SSH server
 
 1. Open a tab with cluster resources by clicking on the plus symbol at the right end of the tab bar.
-   You can also press `Cmd + T` to achieve the same result.
-2. Look for the SSH server you want to connect to and click the Connect button to the right.
-3. Select or enter the SSH user you wish to log in as and press `Enter`.
-4. A new tab will open with a shell session on the chosen server.
+  You can also press `Ctrl/Cmd + T` to achieve the same result.
+1. Look for the SSH server you want to connect to and click the Connect button to the right.
+1. Select or enter the SSH user you wish to log in as and press `Enter`.
+1. A new tab will open with a shell session on the chosen server.
 
-Alternatively, you can type `tsh ssh` into the command bar and the autocompletion will help you
-connect to a server.
+Alternatively, on macOS you can type `tsh ssh` into the command bar and the autocompletion will help
```

Comment:
This works on every platform. Perhaps it wasn't clear that we mean the command bar in Connect?

Reply:
That was indeed my confusion. I'm not a Mac user, and thought that "command bar" was some Mac thingy.

```suggestion
Alternatively, you can type `tsh ssh` into the command bar in the Teleport Connect window, and the autocompletion will help
```